sdk/standbypool/azure-resourcemanager-standbypool/src/main/java/com/azure/resourcemanager/standbypool/fluent/StandbyPoolClient.java

```diff
@@ -1,37 +1,37 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
 
 package com.azure.resourcemanager.standbypool.fluent;
 
 import com.azure.core.http.HttpPipeline;
 import java.time.Duration;
 
 /**
- * The interface for StandbyPoolMgmtClient class.
+ * The interface for StandbyPoolClient class.
  */
-public interface StandbyPoolMgmtClient {
+public interface StandbyPoolClient {
```

Comment:
`title` to be supported in https://github.com/Azure/autorest.java/issues/2487

Reply:
I am not sure who added the "Client", TCGC or us.

If us, one simple solution is to add ManagementClient or MgmtClient when it is ARM.

Anyway, we don't have this class in sample. And this is a preview.